.trunk/trunk.yaml

```diff
@@ -15,6 +15,8 @@ plugins:
 
 lint:
   # enabled list inherited from plugin 'configs'
+  enabled:
+    - osv-scanner@1.3.3
```

Comment:
sanity check: do we want osv-scanner to be enabled in the plugins repo?

Reply:
yeah